### PR TITLE
add optional use_host_network flag

### PR DESCRIPTION
**Pull request by hktudock**

### CODE REVIEW

File: riptide/config/document/command.py [hunk 1]
```diff
@@ -96,6 +96,10 @@ def schema_normal(cls):
             If enabled, read the environment variables in the env-files defined in the project (``env_files``).
             Default: True
 
+        [use_host_network]: bool
+            If enabled, the container uses network mode `host`. Overrides network and port settings
```
Comment (L3P3) [inline riptide/config/document/command.py:100]: *tries to use network... ?

Comment (hktudock) [reply riptide/config/document/command.py:100]: The container will be configured to use network. Maybe he fails but... there is no try ;)

**Review by L3P3 — APPROVED** [review]
No idea if that is correct but no malware seen here. :sweat_smile:

**Review by theCapypara — APPROVED** [review]
Thanks!

**Pull request merged**